extractors/python_functions/smalltalk_extraction/code_snippet.md

```diff
@@ -1,15 +1,14 @@
 ```python
 import re
 from nltk.corpus import stopwords
-from typing import List, Tuple
 
 # currently only english language is supported here
 # reach out to us if you would like to request other languages to be supported
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "smalltalk"
 
-def smalltalk_extraction(record) -> List[Tuple[str, int, int]]:
+def smalltalk_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.